src/datadog.ts

```diff
@@ -1,3 +1,7 @@
 import { DatadogLambda } from "./datadog-lambda";
 
+/**
+ * For backward compatibility. It's recommended to use DatadogLambda for
+ * users who want to add Datadog monitoring for Lambda functions.
+ */
```

Comment:
Could we include the word "Deprecated" in this statement somewhere? This will tell us and users what to expect when we do our next major version update.

Reply:
Sure, will add "To be deprecated.".